_data/changelogs/component-card.yml

```diff
@@ -2,6 +2,13 @@ title: Card
 type: component
 changelogURL:
 items:
+  - date: 2023-11-9
```

Comment:
Other release dates on the 9th of the month have a leading zero 

```suggestion
  - date: 2023-11-09
```

Reply:
Done!